samples/test-samples.cmake

```diff
@@ -50,10 +50,10 @@ if (BUILD_ENCLAVES)
     list(APPEND CRYPTO_LIB_LIST mbedtls)
   endif ()
 
-  if (UNIX)
-    list(APPEND SAMPLES_LIST apkman)
-    list(APPEND CRYPTO_LIB_LIST mbedtls)
-  endif ()
+  # if (UNIX)
```

Comment:
For now can this be disabled only for Jammy?
Also can you explain why this is disabled, along with the error you encountered?